.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,16 @@
+name: PR Build
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+on: pull_request
+
+jobs:
+  call_workflow:
+    name: Run PR Build Workflow
+    if: ${{ github.repository_owner == 'ballerina-platform' }}
+    uses: ballerina-platform/ballerina-library/.github/workflows/pr-build-connector-template.yml@main
+    secrets: inherit
+    with:
+      additional-test-flags: ${{ github.event.pull_request.head.repo.full_name != github.repository && '-x test' || ''}}
```

Comment:
Do we need this additional flag, as we'll be running tests against a mock server anyway?

Reply:
Fixed with [515a334ca674bcae3e1d03d8d0bc355c3be91dd9](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/1/commits/515a334ca674bcae3e1d03d8d0bc355c3be91dd9)